dev/kyuubi-extension-spark-3-1/src/main/resources/sql-classification-default.json

```diff
@@ -1,34 +1,60 @@
 {
   "AlterDatabasePropertiesCommand": "ddl",
-  "AlterTableRenameCommand": "ddl",
-  "AlterTableRenamePartitionCommand": "ddl",
   "AlterTableAddColumnsCommand": "ddl",
-  "AlterTableChangeColumnCommand": "ddl",
   "AlterTableAddPartitionCommand": "ddl",
+  "AlterTableChangeColumnCommand": "ddl",
   "AlterTableDropPartitionCommand": "ddl",
-  "AlterTableSetPropertiesCommand": "ddl",
-  "AlterTableSetLocationCommand": "ddl",
-  "AlterTableSerDePropertiesCommand": "ddl",
-  "AlterTableUnsetPropertiesCommand": "ddl",
+  "AlterTableRecoverPartitionsCommand": "ddl",
+  "AlterTableRenameCommand": "ddl",
   "AlterTableRenameCommand": "ddl",
+  "AlterTableRenamePartitionCommand": "ddl",
+  "AlterTableSerDePropertiesCommand": "ddl",
+  "AlterTableSetLocationCommand": "ddl",
   "AlterTableSetPropertiesCommand": "ddl",
+  "AlterTableSetPropertiesCommand": "ddl",
+  "AlterTableUnsetPropertiesCommand": "ddl",
   "AlterTableUnsetPropertiesCommand": "ddl",
   "AlterViewAsCommand": "ddl",
+  "CreateDataSourceTableAsSelectCommand": "ddl",
+  "CreateDataSourceTableCommand": "ddl",
   "CreateDatabaseCommand": "ddl",
   "CreateFunctionCommand": "ddl",
-  "CreateDataSourceTableCommand": "ddl",
-  "CreateDataSourceTableAsSelectCommand": "ddl",
   "CreateTableLikeCommand": "ddl",
   "CreateViewCommand": "ddl",
   "DropDatabaseCommand": "ddl",
   "DropFunctionCommand": "ddl",
-  "NoopDropTable": "ddl",
   "DropTableCommand": "ddl",
-  "TruncateTableCommand": "ddl",
-  "AlterTableRecoverPartitionsCommand": "ddl",
+  "NoopDropTable": "ddl",
   "SetCatalogAndNamespace": "ddl",
+  "TruncateTableCommand": "ddl",
```

Comment:
TruncateTableCommand

Reply:
There is no such grammar.
In spark, "TRUNCATE TABLE tablename [PARTITION (partcol1=val1, partcol2=val2 ...)]" is belong to "TruncateTableCommand".

Reply:
seems  in spark 3.2

Reply:
we support the user use the self-defined matching rule. if the grammar have added or you want to change some classification for the sql, you can self-defined matching rule with jsonFile named "sql-classification.json". The code will upload your jsonFile.